sdk/python/feast/client.py

```diff
@@ -912,7 +912,7 @@ def get_historical_features(
                     else entity_staging_uri.netloc
                 )
                 staging_client.upload_file(
-                    df_export_path.name, bucket, entity_staging_uri.path
+                    df_export_path.name, bucket, entity_staging_uri.path.lstrip("/")
```

Comment:
shouldn't that `strip` be part of implementation (GS, s3)?

Reply:
We can put `strip` into both implementations too (gcp actually has it already). It just goes a bit against S3 (and i suspect GCP) semantics where `/a` and `a` are different objects

Reply:
I will look into the GCP part again.